testing/dart/canvas_test.dart

```diff
@@ -124,6 +124,19 @@ void testNoCrashes() {
   });
 }
 
+const String kFlutterBuildDirectory = 'kFlutterBuildDirectory';
+
+String get _flutterBuildPath {
+  const String buildPath = String.fromEnvironment(kFlutterBuildDirectory);
+  if (buildPath.isEmpty) {
+    throw StateError('kFlutterBuildDirectory -D variable is not set.');
+  }
+  if (buildPath.startsWith('//')) {
+    return buildPath.substring(2);
+  }
```

Comment:
If you use rebase_path you can drop this.

Reply:
OH so that's what that means! Great, done.

Reply:
Done.